widgets/Ariba-batchsource.json

```diff
@@ -20,7 +20,7 @@
           "label": "API Endpoint",
           "name": "baseURL",
           "widget-attributes": {
-            "placeholder": "Specify the base Path of Ariba API.  E.g. : https://sandbox.api.sap.com"
+            "placeholder": "Specify the base Path of Ariba API, for example, https://sandbox.api.sap.com"
```

Comment:
Change to "Base path of the Ariba API, for example, https://sandbox.api.sap.com."

Reply:
Changed